cdap-client-tests/src/main/java/co/cask/cdap/client/app/AppReturnsArgs.java

```diff
@@ -49,7 +50,8 @@ public void configure() {
     @Path(ENDPOINT)
     @GET
     public void handler(HttpServiceRequest request, HttpServiceResponder responder) {
-      responder.sendJson(200, getContext().getRuntimeArguments());
+      TreeMap<String, String> values = new TreeMap<>(getContext().getRuntimeArguments());
```

Comment:
Just a way to sort the map entries by key. If I just make this change and not modify the test, the test will fail more often.